### ISSUE TITLE
stable.melpa.org seems to be down

### ISSUE THREAD

**Issue by matklad** [melpa/melpa#4640]
Hi! Looks like http://stable.melpa.org/ is down. At least, it does not work in browser, and `package-refresh-contents` hangs if https://stable.melpa.org/packages/ or http://stable.melpa.org/packages/ is in `package-archives`. Plain https://melpa.org/packages/ works fine

**Comment by purcell** [comment]
Indeed, the machine is apparently unreachable, including for ssh access. I imagine that to resolve this, @milkypostman would have to log in to the hosting provider's admin console, sorry.

**Comment by milkypostman** [comment]
I have rebooted the machine. I'm not sure why this happened. The machine (even the admin console) was completely frozen. I am combing through logs, but there is nothing obvious.

I did find where git has been OOMing in the past. Last time was yesterday at 3:30pm PST. It's possible this finally took its toll.  I noticed that it's been pulling HTTP instead of git:// for packages on stable so I changed that and restarted the build. Fingers crossed.
